ui/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="/favicon.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="manifest" href="/manifest.json" />
+    <title>VMClarity</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <script type="module">
```

Comment:
This solution--using inline script here--is going to be a new issue amongst the Solis findings. I know it's a low prior issue, but later we may have to find an other solution to remove them or make them exceptions by adding the hashes.

Reply:
vite transforms it automatically to an external script. You can see it on this screenshot:
<img width="2557" alt="image" src="https://github.com/openclarity/vmclarity/assets/19503754/12778b6f-78ab-4142-ab46-f53019fe4b93">

Reply:
We have this discussed, Vite.js already uses hashes in prod mode, so if we need it later, we can read and use the filename in nginx to get the hash and add it to the CSP header.